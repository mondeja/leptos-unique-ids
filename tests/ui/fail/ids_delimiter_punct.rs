use leptos_unique_ids::leptos_unique_ids;

#[leptos_unique_ids("foo": "bar")]
pub enum Ids {}

fn main() {}
