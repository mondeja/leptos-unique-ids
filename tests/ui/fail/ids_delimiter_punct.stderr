error: Expected a comma between string literals in the attribute.
 --> ui/fail/ids_delimiter_punct.rs:3:1
  |
3 | #[leptos_unique_ids("foo": "bar")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `leptos_unique_ids` (in Nightly builds, run with -Z macro-backtrace for more info)
